### PR TITLE
GCPs and map accuracy

### PR DESCRIPTION
**Pull request by israelbar**
I added information regarding the expected accuracy when using ground control points (GCP). Also included some references for the users to consult.
I hope it can save time for new opendronemap users.

### CODE REVIEW

File: source/arguments.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Arguments::
   --resize-to <integer>
                         resizes images by the largest side for opensfm. Set to
                         -1 to disable.
-			Default: 2048
+                        Default: 2048
```
Comment (danbjoseph) [inline source/arguments.rst:19]: is this an intentional change in spacing?

Comment (israelbar) [reply source/arguments.rst:19]: Totally unintentional. Sorry this is my first time using github.

Comment (danbjoseph) [reply source/arguments.rst:19]: no worries. thanks for contributing, it's great to see people helping out.

Comment (danbjoseph) [reply source/arguments.rst:19]: apologies, i think my initial comment sounded rude. i've had my coffee now. and actually, it looks like you aligned things better than before.

File: source/index.rst [hunk 2]
```diff
@@ -33,6 +33,7 @@ Welcome to OpenDroneMap's documentation
     arguments
     outputs
     gcp
+    GCPaccuracy
```
Comment (danbjoseph) [inline source/index.rst]: there's no standard yet. but i think we might want to change this (and the filename) to `gcp-accuracy`? the others are all lower case and the one with more than 1 word has a hyphen (`requesting-features`).

File: source/arguments.rst (hunk 1 above)
Comment (israelbar) [reply source/arguments.rst:19]: It's OK. The truth is that I was looking into these files trying to understand the way they were written. But after I uploaded the files I realized this file was modified. I couldn't figure out how to revert the changes made and I'm just hoping someone else can solve this in the following steps.

File: source/index.rst (hunk 2 above)
Comment (israelbar) [reply source/index.rst]: Ok, gcp-accuracy sounds good. I can modify this files.

**Comment by danbjoseph** [comment]
thank you. this is all great information that you've added! i think we might want to adjust how it's organized some. 
-  the current docs explain the concept of GCPs with only "Ground control points are useful for correcting distortions in the data and referencing the data to know coordinate systems." you've done a great job expanding on that. instead of a separate page afterwards (as currently  in  the PR), the theory/background details you provide might be better added to that opening  section of the `gcp` file?
-  the "Aspects impacting map accuracy"  that you've added seem to include some more general recommendations that might fit well with the existing info in https://docs.opendronemap.org/tutorials.html#creating-high-quality-orthophotos

@smathermather do you have thoughts on overall docs organization and how to best integrate @israelbar's contributions?

**Comment by israelbar** [comment]
Ok, I can see it a lot clearer.

1. I will move GCP definition and the practices for GCP setting to the "Ground Control Points" section.
2. Now as the GCP information has been relocated, this section would be named only "Map accuracy" and the file renamed to map-accuracy.
3. Regarding on relocating the "Aspects impacting map accuracy" in to the "Creating High Quality Orthophotos", I'm not sure about it, as it is located in the "Tutorials" section.

I will prepare this changes. And And hope you can advice on the number 3.

**Comment by israelbar** [comment]
I think it looks better now...

**Review by smathermather — APPROVED** [review]
It's well written and cohesive as well as a much appreciated contribution.

**Comment by israelbar** [comment]
Thank you so much for giving me the chance to contribute.

**Comment by danbjoseph** [comment]
I approved your request to join the Spanish team on Transifex @israelbar! 👍
Thank you ❤️

**Comment by smathermather** [comment]
> I approved your request to join the Spanish team on Transifex @israelbar! 👍
> Thank you ❤️

Whoa! Fantastic!

**Pull request merged**